diff --git a/components/qiun-data-charts/qiun-data-charts.vue b/components/qiun-data-charts/qiun-data-charts.vue
index a64173fc17977fbd5f2bfeef3d81083935244db5..2e2233376d3f0782a63a437fd2b8f40aac4493c7 100644
--- a/components/qiun-data-charts/qiun-data-charts.vue
+++ b/components/qiun-data-charts/qiun-data-charts.vue
@@ -156,10 +156,10 @@
 </template>
 
 <script>
-import uChartsMp from '@/components/u-charts/u-charts.js';
-import cfu from '@/components/u-charts/config-ucharts.js';
+import uChartsMp from '../u-charts/u-charts.js';
+import cfu from '../u-charts/config-ucharts.js';
 // #ifdef APP-VUE || H5
-import cfe from '@/components/u-charts/config-echarts.js';
+import cfe from '../u-charts/config-echarts.js';
 // #endif
 
 function deepCloneAssign(origin = {}, ...args) {
@@ -1174,9 +1174,9 @@ export default {
 
 <!-- #ifdef APP-VUE || H5 -->
 <script module="rdcharts" lang="renderjs">
-import uChartsRD from '@/components/u-charts/u-charts.js';
-import cfu from '@/components/u-charts/config-ucharts.js';
-import cfe from '@/components/u-charts/config-echarts.js';
+import uChartsRD from '../u-charts/u-charts.js';
+import cfu from '../u-charts/config-ucharts.js';
+import cfe from '../u-charts/config-echarts.js';
 
 var that = {};
 var rootdom = null;
